jenkins-slaves/ansible/data/config/bashrc

```diff
@@ -7,7 +7,9 @@ fi
 
 export LANG=en_US.UTF-8
 export JAVA_HOME=/opt/tools/jdk1.8
-export MAVEN_HOME=/opt/tools/apache-maven-3.2.5
+export MAVEN_HOME=/opt/tools/apache-maven-3.5.2
```

Comment:
I saw somewhere we started to use Maven 3.6.3(?), would it make sense to upgrade to it here as well?

Reply:
Ah yes, I think so. I'll update this to 3.6.3

Reply:
kogito aka quarkus requires at least Maven 3.6.2, so good to use the latest what we have available

Reply:
we should use at least 3.6.0

Reply:
Yes, updated the file to use latest available version(3.6.3) for maven